shared/django_apps/core/migrations/0057_repository_created_at.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.13 on 2024-08-08 19:49
+
+from django.db import migrations, models
+
+"""
+BEGIN;
+--
+-- Add field created_at to repository
+--
+ALTER TABLE "repos" ADD COLUMN "created_at" timestamp with time zone DEFAULT '2024-08-08T21:17:39.913910+00:00'::timestamptz NULL;
+ALTER TABLE "repos" ALTER COLUMN "created_at" DROP DEFAULT;
```

Comment:
that's a really good question. At the moment this will set the value of this column to the default value for all existing rows in the table. I think I will modify this migration to be run such that the command sets the default to null.